object_tracker/object_tracker_test.go

```diff
@@ -385,3 +386,35 @@ func TestInvalidCameraNamesError(t *testing.T) {
 	_, err = fakeTracker.CaptureAllFromCamera(ctx, "", viscapture.CaptureOptions{ReturnImage: true}, emptyMap)
 	test.That(t, err, test.ShouldBeNil)
 }
+
+func TestImageBoundsFromDet(t *testing.T) {
+	bounds := image.Rect(0, 0, 50, 50)
+	det := objdet.NewDetection(bounds, image.Rect(0, 0, 10, 10), 1, LabelDet0)
+	imgBounds := ImageBoundsFromDet(det)
+	test.That(t, *imgBounds, test.ShouldResemble, bounds)
+
+	// Test with no bounds
+	det = objdet.NewDetectionWithoutImgBounds(image.Rect(0, 0, 10, 10), 1, LabelDet0)
+	imgBounds = ImageBoundsFromDet(det)
+	test.That(t, imgBounds, test.ShouldEqual, nil)
+}
+
+func TestReplaceLabelsAndBoundingBox(t *testing.T) {
+	bounds := image.Rect(0, 0, 50, 50)
+	det := objdet.NewDetection(bounds, image.Rect(0, 0, 10, 10), 1, LabelDet0)
+	test.That(t, det, test.ShouldNotBeNil)
+	test.That(t, det.Label(), test.ShouldEqual, LabelDet0)
+	tr := &track{
+		Det: det,
+	}
+
+	newLabel := "dog"
+	replacedTrack := ReplaceLabel(tr, newLabel)
+	test.That(t, replacedTrack.Det.Label(), test.ShouldEqual, newLabel)
+	test.That(t, replacedTrack.Det.NormalizedBoundingBox(), test.ShouldNotBeNil)
```

Comment:
More than not nil, just make sure this is the expected value. Also do the same for the ReplaceBBox test below.